src/main/scala/org/orleans/silo/dispatcher/ClientReceiver.scala

```diff
@@ -17,7 +17,7 @@ import collection.JavaConverters._
 class ClientCleanup(clientSockets: util.List[MessageReceiver])
     extends TimerTask
     with LazyLogging {
-  val CLIENT_REMOVE_TIME_SEC: Int = 10
+  val CLIENT_REMOVE_TIME_SEC: Int = 100000
```

Comment:
Is there a reason you changed it to `100000`? This will keep the client running very long since the server only closes the connection to grains after 100000 seconds.

Reply:
Oh snap, I wanted to test something and forgot to change back. Should be good now.